### PR TITLE
feat: improve answers sections

### PR DESCRIPTION
**Pull request by urielPalomares**

### CODE REVIEW

File: components/game/game.tsx [hunk 1]
```diff
@@ -27,64 +28,72 @@ const GameLayout: NextPage = () => {
     setAnswerSelected(undefined);
   };
 
-  const getButtonState = (answer: string): ButtonStates => {
-    if(!answerSelected) {
+  const getButtonState = (answer: any): ButtonStates => {
```
Comment (Andres2D) [inline components/game/game.tsx:31]: Why any?

File: components/game/game.tsx [hunk 2]
```diff
@@ -27,64 +28,72 @@ const GameLayout: NextPage = () => {
...
+  };
 
-  const answersMap = currentTrack?.answers?.map(answer => 
-    <AnswerButton 
-      key={answer} 
+  const getLyrics = (lyrics: string) => {
+    if (lyrics.length < 351) {
+      return `"${lyrics} ..."`;
+    }
+    return `"${lyrics.substring(0, 350)} ..."`;
```
Comment (Andres2D) [inline components/game/game.tsx:47]: Use .slice instead of substring

File: components/game/game.tsx [hunk 3]
```diff
@@ -27,64 +28,72 @@ const GameLayout: NextPage = () => {
...
+      <Heading textAlign="right" size="xl" mb={2}>
+        {`${gameState.currentSong + 1}/${gameState.playlist.length + 1}`}
+      </Heading>
+      <Heading
+        size="md"
+        textAlign="justify"
+        data-testid="lyrics"
+        mb="5"
+        opacity="0.7"
+        className={styles.lyrics}
```
Comment (Andres2D) [inline components/game/game.tsx:71]: Remove this class.

**Review by Andres2D — APPROVED** [review]

File: components/game/game.tsx (hunk 3 above)
Comment (urielPalomares) [reply components/game/game.tsx]: Ready Sir

**Pull request merged**